postgresql headers include a #define rename(x, y) which conflicts with a
single arg rename() function declaration in Qt headers

--- origsrc/calligra-2.9.2/kexi/kexidb/drivers/pqxx/pqxxconnection.cpp	2015-04-01 08:27:01.000000000 -0500
+++ src/calligra-2.9.2/kexi/kexidb/drivers/pqxx/pqxxconnection.cpp	2015-04-22 09:35:46.965287200 -0500
@@ -17,7 +17,6 @@
  * Boston, MA 02110-1301, USA.
 */
 
-#include "pqxxconnection.h"
 #include <QVariant>
 #include <QFile>
 #include <kdebug.h>
@@ -25,6 +24,7 @@
 #include <db/global.h>
 #include <klocale.h>
 #include <string>
+#include "pqxxconnection.h"
 #include "pqxxpreparedstatement.h"
 #include "pqxxconnection_p.h"
 using namespace KexiDB;
